dali/kernels/imgproc/CMakeLists.txt

```diff
@@ -13,6 +13,7 @@
 # limitations under the License.
 
 add_subdirectory(resample)
+add_subdirectory(colour_manipulation)
```

Comment:
I think `color` is used in our code instead of `colour`

Reply:
+1, regardless of how much of a fan of British English I am.

Reply:
:gb: